include/seqan3/alignment/pairwise/alignment_configurator.hpp

```diff
@@ -262,10 +240,16 @@ struct alignment_configurator
     static constexpr auto configure(config_t const & cfg)
     {
 
-        if constexpr (!config_t::template exists<align_cfg::result>())
+        // If no output is set, everything will set by default.
```

Comment:
Not sure, if you forgot a 'be' or if I'm not familiar with this way of saying it :)
```suggestion
        // If no output is set, everything will be set by default.
```